builder/virtualbox/common/step_port_forwarding.go

```diff
@@ -34,6 +36,58 @@ type StepPortForwarding struct {
 	l *net.Listener
 }
 
+var vboxVerMinNeedLHReachable string
+
+func init() {
+	const vboxVerMinNeedLHReachableNonCanon = "v7.0"
+	var err error
+	vboxVerMinNeedLHReachable, err = getCanonicalSemver(vboxVerMinNeedLHReachableNonCanon)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Plugin failure: %v\nPlease open a Github issue for this plugin.\n", err)
+		os.Exit(1)
+	}
+}
+
+func getCanonicalSemver(version_string string) (string, error) {
```

Comment:
Generally in Go it's preferred to use camelCase rather than snake_case for local/private entities.